Mithril update 2024-06-19 (#411)
IntersectMBO/cardano-updates: blog/2024-06-19-mithril.md

---
title: Mithril Team Update
slug: 2024-06-19-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team continued implementing the certification of Cardano transactions in Mithril networks. They implemented a warmup strategy for the signer and aggregator to begin importing transactions as early as possible and thus minimizing the impact of the activation of their certification by the network. They also significantly improved the throughput of the prover route of the aggregator by fixing some bottlenecks on the database access. Additionally, the team made progress on low-latency certification by completing the handling of rollbacks of the chain and almost completing the retrieval of the transactions with the chain sync mini-protocol and Pallas.
> 
> Finally, they made final modifications on the threat modeling explainer for the documentation website which will be relased shortly and worked on a bug which occurs when parsing some blocks of the SanchoNet network.

Reviewer: ```suggestion
This week, the Mithril team continued implementing the certification of Cardano transactions in Mithril networks. They implemented a warmup strategy for the signer and aggregator to begin importing transactions as early as possible, thus minimizing the impact of network activation on certification. They also significantly improved the throughput of the prover route of the aggregator by addressing database access bottlenecks. Additionally, the team made progress on low-latency certification by completing the handling of chain rollbacks and nearly completing the retrieval of transactions using the chain sync mini-protocol and Pallas.

Finally, they made final modifications to the threat modeling explainer for the documentation website, which will be released shortly, and worked on a bug that occurs when parsing some blocks of the SanchoNet network.
```
